gettsim/transfers/unterhalt.py

```diff
@@ -87,11 +87,14 @@ def unterhaltsvors_m(
     unterhaltsvorschuss_eink = tu_id.replace(unterhaltsvorschuss_eink_tu)
 
     conditions = [
-        (alter < 6) & alleinerziehend,
-        (alter >= 6) & (alter < 12) & alleinerziehend,
+        (alter < list(unterhalt_params["mindestunterhalt"].keys())[0])
```

Comment:
Before creating `conditions`, create a new variable `altersgrenzen = sorted(unterhalt_params["mindestunterhalt"].keys())` and then use that throughout. Safer and more readable.